examples/datasets/plot_limo_data.py

```diff
@@ -33,146 +35,282 @@
+reg = linear_regression(limo_epochs,
+                        design_matrix=design,
+                        names=predictor_vars)
 
 ###############################################################################
-# Visualise effect of phase-coherence.
+# Extract regression coefficients
+# -------------------------------
+#
+# The results stored within the object ``reg``.
+# It basically consists of a dictionary of evoked objects containing
+# multiple inferential measures for each predictor in the design matrix.
+
+print('predictors are:', [key for key in reg.keys()])
+print('fields are:', [field for field in getattr(reg['intercept'], '_fields')])
+
+###############################################################################
+# Plot model results
+# ------------------
+#
+# Now we can access and plot the results of the linear regression analysis by
+# calling `reg['<name of predictor>'].<measure of interest>` and using the
+# `.plot_joint()` method just as we would do with any other evoked object.
+# Below we can see a clear effect of phase-coherence, with higher
+# phase-coherence (i.e., better "face visibility") having a negative effect on
+# the activity measured at occipital electrodes around 200 to 250 ms following
+# stimulus onset.
+
 reg['phase-coherence'].beta.plot_joint(ts_args=ts_args,
-                                       title='Effect of phase-coherence',
-                                       times=[.23])
+                                       title='Effect of Phase-coherence',
+                                       times=[0.23])
 
 ###############################################################################
-# Here we can see a clear effect of phase-coherence, with higher
-# phase-coherence (i.e., better "face visibility") being associated with
-# stronger activity patterns.
+# We can also plot the corresponding T values.
+
+# use unit=False and scale=1 to avoid conversion keep values at their original
+# scale (i.e., avoid conversion to micro-volt).
+ts_args = dict(xlim=(-0.25, 0.5),
+               unit=False)
+topomap_args = dict(scalings=dict(eeg=1),
+                    average=0.05)
+
+fig = reg['phase-coherence'].t_val.plot_joint(ts_args=ts_args,
+                                              topomap_args=topomap_args,
+                                              times=[0.23])
+fig.axes[0].set_ylabel('T-value')
 
 ###############################################################################
 # Conversely, there appears to be no (or very small) systematic effects when
-# constraining Face A and Face B. This is largely consistent with the
+# constraining Face A and Face B stimuli. This is largely consistent with the
 # difference wave approach presented above.
-#
-# Visualise effect of face condition (Face A vs. Face B).
-reg['face a - face b'].beta.plot_joint(title='Contrast: Face A - Face B',
-                                       ts_args=ts_args,
-                                       times=[.15])
+
```

Comment:
```suggestion
ts_args = dict(xlim=(-0.25, 0.5))

```

Reply:
need to re-define this, don't want to apply `unit=False` here.

Reply:
oh, yes, sorry I missed on the previous commit. Thanks!